src/index.ts

```diff
@@ -46,6 +46,8 @@ export { default as useArrowKeyNavigation, type ArrowKeyNavProps } from './hooks
 export { default as useIndexOfLastVisibleChild } from './hooks/useIndexOfLastVisibleChildHook';
 export { default as useIsVisible } from './hooks/useIsVisibleHook';
 export { default as breakpoints } from './utils/breakpoints';
+export { default as ModalPopup } from './Modal/ModalPopup';
+export { default as PopperElement } from './Modal/PopperElement';
```

Comment:
Nit: can you move these a few lines higher up, to keep the list in alphabetical order? e.g. `ModalPopup` should come after `ModalLayer`.